### PR TITLE
Add SMS cheatsheets 

### PR DESCRIPTION
**Pull request by maheshg23**
**What does your Instant Answer do?**
It provides CheatSheet for SMS Language

**What problem does your Instant Answer solve (Why is it better than organic links)?**
It's better because it's a Zero Click Info

**What is the data source for your Instant Answer? (Provide a link if possible)**
Wikipedia - https://en.wikipedia.org/wiki/SMS_language

**Why did you choose this data source?**
Reliable and constantly updated by the community.

**Are there any other alternative (better) data sources?**
Not required.

**What are some example queries that trigger this Instant Answer?**
"sms cheat sheets"

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
Readers 

**Is this Instant Answer connected to a DuckDuckHack Instant Answer idea?**
It's taken from the ideas list provided by @zekiel for a meetup event.

**Which existing Instant Answers will this one supercede/overlap with?**
None

**Are you having any problems? Do you need our help with anything?**
I would like to add few more example queries like "sms cheats" , "sms language" etc .
Its currently not working as I am new to DuckDuckGoHack on IA I do not know how to do this . 
Just learnt this today in the meetup. 

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
screenshot from 2015-07-05 23 05 36
screenshot from 2015-07-05 23 05 57

IA Page: https://duck.co/ia/view/sms_cheat_sheet

### CODE REVIEW

**Comment by MariagraziaAlastra** [comment]
Hi @gmahesh23, great job on this! 
The triggers are the same for all the cheatsheets, you can find the list [here](https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/lib/DDG/Goodie/CheatSheets.pm#L11) - as you can see we have a bunch of them already, so I wouldn't add any more triggers, since it would also affect the other cheatsheets. Thanks!

**Comment by gmahesh23** [comment]
@MariagraziaAlastra  Thank you !!! 
 Yeah i can see . It'll work well with those triggers . But i would suggest "sms Language " as its too specific to the IA . Only if its possible without affecting any other cheatsheets . 
If not, its fine no problem . I'm happy that i could do this much !!

File: share/goodie/cheat_sheets/sms.json [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+            {
+                "key": "ILU",
+                "val": "I Love You"
+            },
+            {
+                "key":"IDK",
+                "val":"I Don't Know"
+            },
+            {
+                "key": "hh  ",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/sms.json]: nit: trailing space here

File: share/goodie/cheat_sheets/sms.json [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+                "key":"BF or BFF",
+                "val":"Best Friend/Boyfriend or Best Friend Forever/Boyfriend Forever"
+            },
+            {
+                "key":"WTH",
+                "val":"What The Hell?"
+            },
+            {
+                "key":"WTF",
+                "val":"What The Fuck?"
```
Comment (moollaza) [inline share/goodie/cheat_sheets/sms.json]: Let's keep this PG:

```
What the #$%@?
```

File: share/goodie/cheat_sheets/sms.json [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+            },
+            {
+                "key":"STFU",
+                "val":"Shut The Fuck Up"
```
Comment (moollaza) [inline share/goodie/cheat_sheets/sms.json]: Same here please

**Comment by gmahesh23** [comment]
@moollaza Thanku.
I have made the changes . Hope its proper now.

File: share/goodie/cheat_sheets/sms.json [hunk 4]
```diff
@@ -0,0 +1,241 @@
+{
+    "id": "sms_cheat_sheet", 
+    "name": "SMS",
+    "description": "Short Message Service (SMS) Language", 
+    "metadata": { 
+        "sourceName": "SMSCheatSheet",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/sms.json]: Please change this to `Wikipedia`

File: share/goodie/cheat_sheets/sms.json [hunk 5]
```diff
@@ -0,0 +1,241 @@
...
+        "sourceURL": "https://en.wikipedia.org/wiki/SMS_language"
+        },
+    "section_order": [  
+        "Frequently Used",
+        "Others",
+        "Abbrivated"
+
```
Comment (moollaza) [inline share/goodie/cheat_sheets/sms.json]: nit: let's remove this extra blank line

File: share/goodie/cheat_sheets/sms.json [hunk 6]
```diff
@@ -0,0 +1,241 @@
...
+    "name": "SMS",
+    "description": "Short Message Service (SMS) Language", 
+    "metadata": { 
+        "sourceName": "SMSCheatSheet",
+        "sourceURL": "https://en.wikipedia.org/wiki/SMS_language"
+        },
+    "section_order": [  
+        "Frequently Used",
+        "Others",
+        "Abbrivated"
```
Comment (moollaza) [inline share/goodie/cheat_sheets/sms.json]: typo: `Abbreviated`

**Comment by moollaza** [comment]
@gmahesh23 thanks a lot for contributing, we really appreciate it! We've left you some feedback that needs to be implemented. Once things are updated please let us know by leaving a comment, so we can review this again. If you have any questions, feel free to ask them here!

In general I'm a little concerned about the sections we're using. I'm not sure they're the best way to organize this info and I think that "others" and "abbreviated" are pretty vague. I think the best approach would be to sort these alphabetically? The Harry Potter Spells Cheat Sheet is a good example

**Comment by gmahesh23** [comment]
@moollaza Thanks for pointing out the mistakes and I have removed the errors as you told .
And have changed the sections and sorted them alphabetically . Do check it .

**Comment by moollaza** [comment]
@gmahesh23 thanks, I've continued this and merged it in the above PR.

**Comment by moollaza** [comment]
@gmahesh23 congrats! It's live: https://duckduckgo.com/?q=sms+cheat+sheet&ia=cheatsheet

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's, tackle open issues or even submit more Instant Answers!

**Comment by gmahesh23** [comment]
Thanks a lot :)
will be working on more from now on .
Its really fun to contribute to DDG.

**Comment by moollaza** [comment]
>Thanks a lot :) 

No Problem! Thanks for contributing!

>will be working on more from now on . Its really fun to contribute to DDG.

Awesome :+1:  We look forward to it :smiley:

**Pull request merged**